src/data-workspace/data-workspace.test.js

```diff
@@ -0,0 +1,95 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+import { useWorkflowContext } from '../workflow-context/index.js'
+import { useSelectionParams } from '../workflow-context/use-selection-params.js'
+import { DataSetNavigation } from './data-set-navigation/index.js'
+import { DataWorkspace } from './data-workspace.js'
+import { Display } from './display/index.js'
+import { TitleBar } from './title-bar/index.js'
+
+jest.mock('../workflow-context/index.js', () => ({
+    useWorkflowContext: jest.fn(),
+}))
+
+jest.mock('../workflow-context/use-selection-params.js', () => ({
+    useSelectionParams: jest.fn(),
+}))
+
+const mockDataSets = [
+    {
+        id: 'data-set-1',
+        displayName: 'Data set 1',
+        periodType: 'Monthly',
+    },
+    {
+        id: 'data-set-2',
+        displayName: 'Data set 2',
+        periodType: 'Monthly',
+    },
+]
+
+beforeEach(() => {
+    useWorkflowContext.mockImplementation(() => ({
+        displayName: 'Workflow a',
+        id: 'i5m0JPw4DQi',
+        periodType: 'Monthly',
+        dataSets: mockDataSets,
+        approvalState: 'APPROVED_HERE',
+    }))
+    useSelectionParams.mockImplementation(() => ({
+        pe: '201204',
+        ou: 'ImspTQPwCqd',
+    }))
+})
```

Comment:
There's no need for the `beforeEach`. `mockImplelentationOnce` will restore the original mock after it's been called once, so this basically sets the default behavior